src/blogtoblog/BlogToBlogImporter.ts

```diff
@@ -125,18 +123,26 @@ export default class BlogToBlogImporter extends PageImporter {
       const categoryData = document.createElement('td');
       categoryData.textContent = category;
       categoryRow.append(categoryData);
-      if (topics.length) {
+      topics = topicsArr.join(', ');
+      if (topicsArr[0] !== 'undefined') {
```

Comment:
How can this be ? Having a string with `undefined` value is a bad practice, we should catch this earlier.

Reply:
If the products and topics only have one item between then, that item will become the category, no longer a product or topic. But agreed and fixed.